src/os/bluestore/BlueStore.cc

```diff
@@ -9691,7 +9691,7 @@ int BlueStore::_generate_read_result_bl(
     const BlobRef& bptr = b2r_it->first;
     regions2read_t& r2r = b2r_it->second;
     dout(20) << __func__ << "  blob " << *bptr << std::hex
-             << " need 0x" << r2r << std::dec << dendl;
+             << " need " << r2r << std::dec << dendl;
```

Comment:
i think we can go further by removing the stream manipulators around `r2r`. i.e. to remove `std::hex` and `std::dec` here.

Reply:
yeah, it's great.